apps/google-analytics-4/frontend/src/config.ts

```diff
@@ -12,8 +12,13 @@ function getEnvironmentVariable(
   return environmentVariableValue;
 }
 
+const backendApiUrl =
+  getEnvironmentVariable('REACT_APP_ENV', 'prod') === 'test'
```

Comment:
REACT_APP_ENV is set to `test` in circle CI in the apps-staging context so that we can determine which URL to use for API calls.